rllib/agents/dqn/tests/test_simple_q.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
```

Comment:
Added this just for SimpleQ (includes a loss function test).
Test for all tf/eager/torch.